reporters/ReportVectorMigration.cpp

```diff
@@ -166,33 +227,60 @@ GetReportInstantiator( Kernel::instantiator_function_t* pif )
             throw QueryInterfaceException(__FILE__, __LINE__, __FUNCTION__, "pvc", "IMigrate", "IVectorCohort");
         }
 
+        VectorStateEnum::Enum state = pvc->GetState();
+        if (!m_MustBeInState.empty())
+        {
+            if (!std::count(m_MustBeInState.begin(), m_MustBeInState.end(), state)) 
```

Comment:
FYI. Normally, we use std::find() for something like this.  It would be faster if the list were very long.  In this, this is fine and not change is required.